File: 0292_Nim_Game/292.cpp
#include <iostream>
using namespace std;
class Solution
{
public:
    bool canWinNim(int n)
    {
        return n % 4 != 0;
    }
};
int main()
{
    Solution a;
    cout << a.canWinNim(4) << " " << a.canWinNim(1) << " " << a.canWinNim(2) << " ";
    return 0;
}

File: 0292_Nim_Game/292.py
class Solution:
    def canWinNim(self, n: int) -> bool:
        return n % 4 != 0

a = Solution()
print(a.canWinNim(n = 4))
print(a.canWinNim(n = 1))
print(a.canWinNim(n = 2))